desktop/src/main/java/bisq/desktop/util/GUIUtil.java

```diff
@@ -700,18 +700,18 @@ public static String getPercentage(Coin part, Coin total) {
         return t.cast(parent);
     }
 
-    public static void showTakeOfferFromUnsignedAccountWarning(CoinFormatter coinFormatter) {
+    public static void showTakeOfferFromUnsignedAccountWarning(CoinFormatter coinFormatter, Coin maxLimit) {
         String key = "confirmTakeOfferFromUnsignedAccount";
-        new Popup().warning(Res.get("payment.takeOfferFromUnsignedAccount.warning", coinFormatter.formatCoinWithCode(OfferRestrictions.TOLERATED_SMALL_TRADE_AMOUNT)))
+        new Popup().warning(Res.get("payment.takeOfferFromUnsignedAccount.warning", coinFormatter.formatCoinWithCode(maxLimit)))
                 .width(900)
                 .closeButtonText(Res.get("shared.iConfirm"))
                 .dontShowAgainId(key)
                 .show();
     }
 
-    public static void showMakeOfferToUnsignedAccountWarning(CoinFormatter coinFormatter) {
+    public static void showMakeOfferToUnsignedAccountWarning(CoinFormatter coinFormatter, Coin maxLimit) {
         String key = "confirmMakeOfferToUnsignedAccount";
-        new Popup().warning(Res.get("payment.makeOfferToUnsignedAccount.warning", coinFormatter.formatCoinWithCode(OfferRestrictions.TOLERATED_SMALL_TRADE_AMOUNT)))
+        new Popup().warning(Res.get("payment.makeOfferToUnsignedAccount.warning", coinFormatter.formatCoinWithCode(maxLimit)))
                 .width(900)
                 .closeButtonText(Res.get("shared.iConfirm"))
                 .dontShowAgainId(key)
```

Comment:
The message here (`showMakeOfferToUnsignedAccountWarning` and `showTakeOfferFromUnsignedAccountWarning`, called from a few places in this PR) is:
> With the recent rise in XMR price, beware that selling {0} or less incurs higher risk than before.\n\n\It is highly recommended to either:\n\- make offers &gt;{0}, so you only deal with signed/trusted buyers\n\- keep any offers to sell &lt;{0} to around ~100 USD in value, as this value has (historically) discouraged scammers

and it had 0.01 BTC hardcoded in Bisq.


I think here it may be always the TOLERATED_SMALL_TRADE_AMOUNT value, which I believe is never reduced (if the account isn't signed etc)

Reply:
Ok I reverted this change.